data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report6_fitdata.py

```diff
@@ -16,7 +16,7 @@
 #
 
 import pandas as pd
-from common import JINJA_ENV, FITBIT_TABLES
+from common import JINJA_ENV, FITBIT_TABLES, PIPELINE_TABLES, SITE_MASKING_TABLE_ID
 from utils import auth
```

Comment:
Pipeline tables is a notebook parameter. Would you use the parameter instead so that it is consistent throughout the notebook.